merge.cpp

```diff
@@ -46,7 +69,116 @@ void LoadSampleSets(const int rank, const double energyFraction, const int nsets
     {
         cout << varName << " basis summary output: ";
         BasisGeneratorFinalSummary(basis_generator.get(), energyFraction, cutoff);
-        PrintSingularValues(rank, basename, varName, basis_generator.get(), usingWindows, window);
+        PrintSingularValues(rank, basename, varName, basis_generator.get(), usingWindows, basisWindow);
+    }
+}
+
```

Comment:
@siuwuncheung Could you please add some documentation explaining the overall algorithm for merging parametric bases when defined by number of samples or physical time, as well as a description of what these new functions do in that algorithm? It is not very easy to see the algorithm from the code.